sugar-slug/Utils/RigidBodyUtils.cs

```diff
@@ -0,0 +1,71 @@
+﻿using sugar_slug.Engine.Physics;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace sugar_slug.Utils {
+    class RigidBodyUtils {
+        public RigidBodyUtils() {
+
+        }
+        public static Vector GetCenter(RigidBody body) {
+            return (body.Size / 2);
```

Comment:
The "Center" position can't be relative to the Body itself, otherwise RigidBodies of the same size would always have the same Center position.

```suggestion
            return new Vector(
                body.Position.X + (body.Size.X / 2),
                body.Position.Y + (body.Size.Y / 2),
                body.Position.Z
            );
```

Reply:
GetMiddle should be deleted then.